src/training/communicator.cpp

```diff
@@ -139,7 +139,7 @@ class FakeMPIWrapper : public IMPIWrapper
 {
 public:
   FakeMPIWrapper(bool) {
-    LOG(warn, "Compiled without MPI support. Falling back to FakeMPIWrapper");
+    LOG(debug, "[comm] Compiled without MPI support. Falling back to FakeMPIWrapper");
```

Comment:
Also, let's keep info.

Reply:
Ah, original had "warn", but I think "info" is OK here.

Reply:
I followed Frank's suggestion with "Running as single process on ..."